core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -21,13 +21,26 @@
  */
 public class ClientConfigProperties extends AbstractConfig {
 
+    /**
+     * The default amount of time to wait for credits after link creation.
+     */
+    public static final long DEFAULT_FLOW_LATENCY = 20L; //ms
```

Comment:
Did you changed this on purpose? If yes, do think 20 is a better value?

Reply:
Running locally on my machine I still experienced 503 results using 10ms. Do you think 20ms is too much for a default?

Reply:
No, I was only interested in your thoughts - on my machine 10ms worked, but than the default of 20ms might be better.